requirements.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: airflow
-  repository: https://github.com/astronomer/airflow/releases/download/helm-chart%2Fv1.0.1-dev1/
+  repository: https://github.com/astronomer/airflow/releases/download/oss-helm-chart%2F1.1.0-rc1/
```

Comment:
not sure what this change is from. Should I try w/o it?

Reply:
Yeah, let's use that change

Reply:
Yeah, that's the latest snapshot of the OSS chart. We definitely need it.

When the `1.1.0` chart lands, this will switch to `https://airflow.apache.org` (most likely).